packages/taquito/src/contract/interface.ts

```diff
@@ -273,6 +275,16 @@ export interface ContractProvider extends StorageProvider {
    * @param SmartRollupOriginateParams smartRollupOriginate operation parameter
    */
   smartRollupOriginate(params: SmartRollupOriginateParams): Promise<SmartRollupOriginateOperation>;
+  /**
+   * @description Execute an outbox message of the specified smart rollup
```

Comment:
slightly nitpick would be great to use  octez-client description `Execute a message from a smart rollup's outbox of a cemented commitment`. Also applies to other files